src/Microsoft.Win32.Primitives/src/Microsoft.Win32.Primitives.csproj

```diff
@@ -28,7 +32,7 @@
     </Compile>
     <Compile Include="System\ComponentModel\Win32Exception.Windows.cs" />
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' And '$(TargetGroup)' == 'netcoreapp' ">
+  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' And ( '$(TargetGroup)' == 'netcoreapp'  Or '$(TargetGroup)' == 'uap' ) ">
```

Comment:
yup, once I remove all uap configs that are not windows specific I should be able to clean these changes up.